src/test/java/teammates/test/cases/ui/InstructorFeedbackQuestionAddActionTest.java

```diff
@@ -635,204 +618,186 @@ public void testExecuteAndPostProcess() throws Exception{
+        modifyParamValue(params, Const.ParamsNames.FEEDBACK_QUESTION_RECIPIENTTYPE, "INSTRUCTORS");
+
         action = getAction(params);
         result = (RedirectResult) action.executeAndPostProcess();
-        
-        assertEquals(
-                Const.ActionURIs.INSTRUCTOR_FEEDBACK_EDIT_PAGE
-                        + "?courseid="
-                        + instructor1ofCourse1.courseId
-                        + "&fsname=First+feedback+session"
-                        + "&user="
-                        + instructor1ofCourse1.googleId
-                        + "&error=false",
-                result.getDestinationWithParams());
-        
+
+        assertEquals(Const.ActionURIs.INSTRUCTOR_FEEDBACK_EDIT_PAGE + "?courseid=" + instructor1ofCourse1.courseId +
+                     "&fsname=First+feedback+session" + "&user=" + instructor1ofCourse1.googleId + "&error=false",
+                     result.getDestinationWithParams());
+
         assertEquals(Const.StatusMessages.FEEDBACK_QUESTION_ADDED, result.getStatusMessage());
-        
-        expectedLogMessage =
-                "TEAMMATESLOG|||instructorFeedbackQuestionAdd|||instructorFeedbackQuestionAdd|||true|||"
-                        + "Instructor|||Instructor 1 of Course 1|||idOfInstructor1OfCourse1|||instr1@course1.tmt|||"
-                        + "Created Feedback Question for Feedback Session:<span class=\"bold\">"
-                        + "(First feedback session)</span> for Course <span class=\"bold\">[idOfTypicalCourse1]</span>"
-                        + " created.<br><span class=\"bold\">Essay question:</span> question|||/page/instructorFeedbackQuestionAdd";
+
+        expectedLogMessage = "TEAMMATESLOG|||instructorFeedbackQuestionAdd|||" +
+                             "instructorFeedbackQuestionAdd|||true|||" +
+                             "Instructor|||Instructor 1 of Course 1|||" +
+                             "idOfInstructor1OfCourse1|||instr1@course1.tmt|||" +
+                             "Created Feedback Question for Feedback Session:<span class=\"bold\">" +
+                             "(First feedback session)</span> for Course " +
+                             "<span class=\"bold\">[idOfTypicalCourse1]</span>" +
+                             " created.<br><span class=\"bold\">Essay question:</span> " +
+                             "question|||/page/instructorFeedbackQuestionAdd";
         assertEquals(expectedLogMessage, action.getLogMessage());
-        
+
         ______TS("Failure: Empty or null participant lists");
-        
+
         params = createParamsForTypicalFeedbackQuestion(fs.courseId, fs.feedbackSessionName);
-        params[21] = ""; //change showGiverTo to empty
-        
-        // remove showRecipientTo
+        modifyParamValue(params, Const.ParamsNames.FEEDBACK_QUESTION_SHOWGIVERTO, "");
+
+        // Purposefully not using modifyParamVale because we're removing showRecipientTo
```

Comment:
I might need to take a look at that again and make sure it is done purposely, I forgot whether I counted right, but yeah purposely :p